### DISCUSSION TITLE
Run bash scripts

### DISCUSSION THREAD

**Discussion by zebede22** [nushell/nushell#12214] [Q&A] [answered]
App installers usually provide bash scripts to setup installation,  take brew for example:
```bash
/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
```

it fails.

**Comment by ghost** [comment, marked as answer]
this command uses bash syntax for string substitution - its usually easiest to just start bash (`bash`) and then run the command in there.

nushell does not try to be compatible with bash code at all.

if you wanted to run this command in nushell you would have to reformat it to this:
```nu
bash -c (curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)
```
